iot_packet_verifier/src/lib.rs

```diff
@@ -0,0 +1,127 @@
+use futures::stream::StreamExt;
+use slog::{self, info, o, Drain, Level, Logger};
+use std::collections::HashMap;
+//use std::io::{ErrorKind, Result};
+//use tokio_stream::StreamExt;
+// use tokio::sync::mpsc;
+
+use helium_proto::services::router::PacketRouterPacketReportV1;
+use helium_proto::Message;
+use poc_store::{file_source, Result};
+// FIXME for writing to S3:
+//use poc_store::FileStore;
+
+// Inputs:
+type GatewayPubKey = Vec<u8>;
+#[allow(clippy::upper_case_acronyms)]
+type OUI = u32;
+
+// Outputs:
+// type Bookkeeping = helium_proto::IotPacketVerifierBookkeeping;
+// type OuiBooks = helium_proto::OuiBookkeeping;
+// type GatewayBooks = helium_proto::GatewayBookkeeping;
+
+// While processing:
+
+pub struct Counts {
+    pub gateway: GatewayPubKey,
+    pub oui: OUI,
+}
+
+// TODO move to Settings.toml
```

Comment:
Not from settings.toml. This ain’t gateway-rs. Look at the other components on how dotenv is used

Reply:
Comment removed in commit e0e1c6a.  (There was no code for settings.toml yet.)